R/plot_xctr_company_level.R

```diff
@@ -0,0 +1,48 @@
+#' Plot xctr data on a company level
+#'
+#' @param data A data frame like [xctr_toy_data].
+#' @param company_name A string. Name of one company in the data set.
+#'
+#' @return A [ggplot] object.
+#' @export
+#'
+#' @examples
+#' plot_xctr_company_level(xctr_toy_data, "company_a")
+plot_xctr_company_level <- function(data, company_name) {
+
+  data <- data |>
+    na.omit() |>
```

Comment:
Seems a bit dangerous as this would drop `NA`s everywhere and silently?
See how ggplot2 does it. Whenever it drops `NA`s you get a warning and the scope is usually limited to one specific column.